_attendees/BismarkFrimpong.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+name: Bismark Frimpong
+position: Student (Adiminstrative Support)
```

Comment:
there is a minor typo here 😄

Reply:
the hand is faster than the mind 😄, typo fixed